### PR TITLE
Improvement/cx 6912 upgrade react phone number input

### PR DESCRIPTION
**Pull request by kopijunkie**
# Task
Web SDK is still on v2 of `react-phone-number-input` and there are some NPM vulnerabilities, React version conflicts that require upgrading to v3.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by josokinas — APPROVED** [review]

**Comment by kopijunkie** [comment]
> I can see that `libphonenumber-js` is back on the `onfido.min.js` bundle. It should only be in crossDevice bundle to prevent unnecessary size increase of the main bundle. You can compare the [report](https://1602-pr-onfido-sdk-ui-onfido.surge.sh/reports/bundle_idv_dist_size.html) from this change with the [report for latest](https://latest-onfido-sdk-ui-onfido.surge.sh/reports/bundle_idv_dist_size.html).
> 
> The screenshot below shows the duplication of `libphonenumber-js` in red.
> 
> Screenshot 2021-11-09 at 12 54 07

Unfortunately it looks like this is a side effect of upgrading to v3 of this library. The architecture has been changed in a way that the `getCountryCodes(labels)` method we were using has been removed in favour of this `getCountries()` method. Additionally, contrary to their [CHANGELOG](https://gitlab.com/catamphetamine/react-phone-number-input/-/blob/master/CHANGELOG.md), `getCountries()` actually requires a `metadata` object which only exists in `libphonenumber-js`. 

We should have a chat when you're back about the impact of this change or pair to see if there is a better workaround I've not tried yet.

File: src/index.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ const experimentalFeatureWarnings = ({ steps }: NormalisedSdkOptions) => {
 const isSMSCountryCodeValid = (smsNumberCountryCode: string) => {
   // If you need to refactor this code, remember not to introduce large libraries such as
   // libphonenumber-js in the main bundle!
-  const countries = getCountryCodes(labels)
+  const countries = getCountries(metadata) as string[] // react-phone-number-input returns an array of CountryCode type contrary to documentation
```
Comment (stefaniacardenas) [inline src/index.tsx:85]: Is this the only way of doing this validation? I don't like the idea of importing this large library globally for this validation only tbh...

Comment (kopijunkie) [reply src/index.tsx:85]: Agreed about having to introduce the large library just for the country code validation. I've had a think & what I can try is hacky, but we could make use of the `labels` value to do the validation like it used to. The file it was originally imported from still exists, just renamed/restructured.

File: package.json [hunk 2]
```diff
@@ -93,15 +93,15 @@
     "files": [
       {
         "path": "./dist/onfido.min.js",
-        "maxSize": "240 kB"
+        "maxSize": "260 kB"
```
Comment (stefaniacardenas) [inline package.json:96]: Do we still need to bump this?

Comment (kopijunkie) [reply package.json:96]: Good point. I'll revert this change.

**Review by stefaniacardenas — APPROVED** [review]

**Pull request merged**